pkg/azure/create.go

```diff
@@ -266,6 +267,22 @@ func createVirtualMachine(
 		return nil, err
 	}
 
+	if azureProvider.Config.CustomData != "" {
```

Comment:
It would be nice to support either a path to a file, or a base64 string already

You could check if the file does not exists, if it also decodes as a valid b64, we could use that